corpus/data_augmentation_audiomentations.py

```diff
@@ -0,0 +1,127 @@
+__all__ = ["AddImpulseResponseJob", "MixNoiseJob"]
+
+import os
+import audiomentations as am
+import shutil
+import logging
+from scipy.io import wavfile
+
+
+from sisyphus import *
+from recipe.i6_core.lib import corpus
+
+class AddImpulseResponseJob(Job):
+    def __init__(self, corpus_file, new_corpus_name, ir_path, p):
+        self.corpus_file = corpus_file
+        self.new_corpus_name = new_corpus_name
+        self.impulse_responses_dir = ir_path 
+        self.p_impulse_response = p
+
+        self.out_audio_folder = self.output_path("out_audio/", directory=True)
+        self.out_corpus = self.output_path("ir_convoluted.corpus.xml.gz")
+        self.out_segment_file = self.output_path("ir_convoluted.segments")
+
+        self.rqmt = {"time": 12, "cpu": 2}
+
+        self.add_impulse_response = am.AddImpulseResponse(ir_path=ir_path, p=p)
+
+    def tasks(self):
+        yield Task("run", rqmt=self.rqmt)
+
+    def run(self):
+        self.id = os.path.basename(self.job_id())
+        if not os.path.isdir(f"/dev/shm/{self.id}"):
+            os.mkdir(f"/dev/shm/{self.id}")
+        c = corpus.Corpus()
+        nc = corpus.Corpus()
+        segment_file_names = []
+
+        c.load(tk.uncached_path(self.corpus_file))
+        nc.name = self.new_corpus_name
+        nc.speakers = c.speakers
```

Comment:
```suggestion
        nc.speakers = list(c.all_speakers())
```
maybe we want to use `all_speakers()` to capture speakers defined in subcorpora?